content/en/docs/reference/kube-scheduler/_index.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Kube-Scheduler
```

Comment:
```suggestion
title: Scheduling
```
(and change the URL to match)

I'm suggesting this for future-proofing againstchange.
If the `kube-scheduler` is replaced with two components `kube-schedule-filter` and `kube-schedule-scorer`, the reference docs section can keep its URL.

Reply:
Done